packages/webcert/src/store/fmb/fmbMiddleware.ts

```diff
@@ -118,7 +188,7 @@ function retrieveFMBForAddedDiagnosisCodes(
 ) {
   valueDiagnosisList.list.forEach((diagnosis, index) => {
     const exists = existingFMBDiagnosisCodeInfo.findIndex((existing) => existing.icd10Code === diagnosis.code) > -1
-    if (exists) {
+    if (exists || !diagnosis.code || diagnosis.code == '') {
```

Comment:
isn't it enough with !diagnosis.code as it will be falsy if the string is empty. 

But if I'm mistaken, then consider to use === instead of ==.